### ISSUE TITLE
Suggestion: Inform Users Regarding Quotation Marks

### ISSUE THREAD

**Issue by amnotbananaama** [TheEssem/Image2Map#1]
Thanks for creating this mod! Would it be possible for you to add documentation that paths containing a `/`, `\`, or `:` (in other words, if your image file is NOT in the root directory of your server folder) need to be [escaped](https://github.com/Mojang/brigadier/blob/559d8f3/src/main/java/com/mojang/brigadier/StringReader.java#L169) with single or double quotation marks? I was not familiar with this part of Brigadier, and I would imagine less technical users will be even more confused. Thanks!

**TL;DR**

`/mapcreate example.jpg` **VALID**
`/mapcreate "example.jpg"` **VALID**
`/mapcreate 'example.jpg'` **VALID**
`/mapcreate "https://example.com/example.jpg"` **VALID**
`/mapcreate "examplefolder/example.jpg"` **VALID**

`/mapcreate https://example.com/example.jpg` **INVALID**
`/mapcreate examplefolder/example.jpg` **INVALID**

**Comment by TheEssem** [comment]
I could probably get around to documenting that, a more ideal solution would be to try and make it so that the quotes aren't needed however

**Comment by TheEssem** [comment]
Quotes are no longer needed as of e1f171a.
